WcaOnRails/app/models/user_group.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class UserGroup < ApplicationRecord
+  has_many :child_groups, class_name: "UserGroup", foreign_key: "parent_group_id"
```

Comment:
Nitpick 1: Move this down to the other associations like `metadata` and `parent_group`.
Nitpick 2: Use `inverse_of: :parent_group`, as per my advice to check out `stripe_transaction.rb`

Reply:
Done